targets/unity-v2/source/Shared/Public/Pipeline/OneDSEventPipeline.cs

```diff
@@ -76,11 +79,19 @@ public Task PipelineTask
         public OneDSEventPipeline(OneDSEventPipelineSettings settings, ILogger logger)
         {
             if (settings == null)
```

Comment:
Can we ensure these are surrounded with brackets { } ?

Would make adding anything in the future easier if necessary.